plugins/aws/s3/config/plugin-config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "app": {
+        "name": "hopeit-aws-s3",
+        "version": "${HOPEIT_APPS_API_VERSION}"
+    },
+    "engine": {
```

Comment:
ObjectStorage settings should be here, not in app config file.

Reply:
Or this file can be removed?

Reply:
removed